lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -16,9 +16,8 @@ def munge!
         # This is the good stuff.
         site.config["github"] = github_namespace
 
-        return unless should_add_fallbacks?
-        add_url_and_baseurl_fallbacks!
         add_title_and_description_fallbacks!
```

Comment:
I wonder if we should bail early if:

1. No auth token is present (if they rebuild frequently, they'll start getting 403 Rate Limit Exceeded errors which are confusing)
2. There is no internet connectivity